riotfile.py

```diff
@@ -447,7 +447,7 @@ def select_pys(min_version=min(SUPPORTED_PYTHON_VERSIONS), max_version=max(SUPPO
                         "pytest-asyncio": latest,
                         "requests": latest,
                         "aiofiles": latest,
-                        "sqlalchemy": latest,
+                        "sqlalchemy": "==1.3",
```

Comment:
```suggestion
                        "sqlalchemy": "~=1.3.0",
```

?

Do we explicitly want `1.3.0` only?

Reply:
ah yeah for some reason I was thinking `==1.3` would do `>=1.3,<1.4`

Reply:
done!